ee/webhooks/build.Dockerfile

```diff
@@ -1,5 +0,0 @@
-FROM ghcr.io/formancehq/base:22.04
```

Comment:
This file is used by the CI to build releases

Reply:
I restored it.